inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -83,6 +85,520 @@ std::string name(const Node& n) {
     return n->get_friendly_name();
 }
 
+namespace attr_comparison {
+
+using AttrName = std::string;
+
+class Result {
+public:
+    explicit Result(std::string m = {}) : m_message(std::move(m)) {}
+
+    const std::string& message() const {
+        return m_message;
+    }
+
+    bool has_error() const {
+        return !m_message.empty();
+    }
+
+    Result& operator+=(const std::string& msg) {
+        m_message.append(m_break_line_no, '\n').append(msg);
+        m_break_line_no = 1;
+        return *this;
+    }
+
+private:
+    std::string m_message;
+    int m_break_line_no{0};
+};
+
+using SubGraphOpInputDescription =
+    std::vector<std::shared_ptr<ngraph::op::util::SubGraphOp::InputDescription>>;
+
+using SubGraphOpOutputDescription =
+    std::vector<std::shared_ptr<ngraph::op::util::SubGraphOp::OutputDescription>>;
+
+using SpecialBodyPorts = ngraph::op::v5::Loop::SpecialBodyPorts;
```

Comment:
ngraph::op::v5 -> ngraph::opset6
@GlebKazantaev we had such an agreement, right?

Reply:
Yes, right.

Reply:
Will be address in 47842.